### PR TITLE
created new features in pTables.py 

### PR DESCRIPTION
**Pull request by JoshRupp**
New Features include:  dynamic column size calculation and text wrapping per column. Also included in this push is
table row dividers and a simple non complete test package of my pTables.py
file.

### CODE REVIEW

File: Information/Design.md [hunk 1]
```diff
@@ -310,7 +354,7 @@ examples of _-edit_  command usage:
 ------------------------------------------------------------------------------------------
 
 
-######REQ 160 Design:
+######REQ 160 Design:  Josh
```
Comment (EnderStir) [inline Information/Design.md]: Missing the word 'by' that all of the others have.

**Review by EnderStir — APPROVED** [review]
All file look fine. The minor issue with the Design.md file can be ignored. Now that these tests are added, we still need a coverage report.

File: Information/Design.md (hunk 1 above)
Comment (JoshRupp) [reply Information/Design.md]: Ok I will fix it

Comment (JoshRupp) [reply Information/Design.md]: Ok will fix it

Comment (JoshRupp) [reply Information/Design.md]: fixed it

**Pull request merged**